src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -35,6 +35,14 @@
     crafted: true
 """
 
+helps['backup container unregister'] = """
+type: command
+short-summary: This command allows Azure Backup to unregister a ‘Backup Container’ so that the underlying 'resource' can be protected to another vault, if required. The existing backup data in this vault should be deleted before 'unregister' can be performed.
```

Comment:
Short-summary message will be displayed when users enter `az backup container -h`. You short-summary message here is too long and we usually use active voice for short summary. Please change to something like "Unregister a Backup Container to make the underlying  'resource' be protected by another vault". Just in the same format, you can define your message by yourself.
The same for examples name.